src/qt/sendcoinsentry.cpp

```diff
@@ -250,13 +250,13 @@ bool SendCoinsEntry::updateLabel(const QString &address)
     if(!model)
         return false;
 
-    // Fill in label from address book, if address has an associated label
+    // Fill in label from address book
     QString associatedLabel = model->getAddressTableModel()->labelForAddress(address);
-    if(!associatedLabel.isEmpty())
+    bool res = QString::compare(ui->addAsLabel->text(), associatedLabel, Qt::CaseSensitive) != 0;
+    if(res)
     {
         ui->addAsLabel->setText(associatedLabel);
-        return true;
     }
 
-    return false;
+    return res;
```

Comment:
at this point, `res` can only be `false`. Changing line 261 makes no sense to me.